### ISSUE TITLE
[Req] Master settings for Insights logging

### ISSUE THREAD

**Issue by Radiotechniman** [athombv/homey#2726]
**Metadata**

* **Homey Version**: 1.5.11
* **Homey Log Code**: N/a
* **Smartphone Platform**: N/a
* **Smartphone App Version**:N/a

**What did you try to achieve?**

Reducing the load on homey with all the insights gathering i'd like to be able to control the # of insights and the duration of the log time

**What did you expect as result?**

I'm expecting a setting in the Homey "homey:manager:system" where I can say "Log nothing", "Log for XX days" etc. 
Also would be nice to be able to set the default logging action and finetune this per entry (e.g. my default insight logging action would be "Log for 3 days" but I can then fine tune the individual entries to log more) Similarly setting defaults per insight type, e.g. Logging of power use (Watt) could be 60 days but the log of Voltage (220v) would be set "do not log"  

**What was the result?**

this needs to be done per item, and per sub item which is a lot of work and you easily miss one or two

**Any other remarks**

**Comment by Dijker** [comment]
Good point! I hope we get more options like mentioned in Homey 2.0 / the Mobile App. 
You know you can already remove individual values you don't want?
![image](https://user-images.githubusercontent.com/12412734/43516448-ebf5ccce-9585-11e8-811b-811e9f5f9d18.png)

**Comment by Radiotechniman** [comment]
Yes i do, but i have so many that it's too much work to do that, just look at your example, 
That device already has 5 things that are generating insights
Imagine how easy it would be if you can say in the general settings "do not generate insights for <battery> on any device"  

Thinking about it, might even be a great use-case to do this in a flow, e.g. allow a user to mark thing to generate (or not generate) insights through the flow editor
